### ISSUE TITLE
Fix epytext markup issues in twisted.words.xish

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5555]
|<img alt="ClearliteX's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| ClearliteX reported|
|-|-|
|Trac ID|trac#5555|
|Type|enhancement|
|Created|2012-03-16 00:07:26Z|
|Branch|https://github.com/twisted/twisted/tree/xish-epytext-5555|

We have a ton of invalid epytext that produces warnings like shown in http://pastebin.com/Hd7F0Z0G when we generate the docs.

<details><summary>Searchable metadata</summary>

```
trac-id__5555 5555
type__enhancement enhancement
reporter__ClearliteX ClearliteX
priority__normal normal
milestone__ 
branch__branches_xish_epytext_5555 branches/xish-epytext-5555
branch_author__ralphm ralphm
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1331856446000000 1331856446000000
changetime__1331925402000000 1331925402000000
version__None None
owner__ralphm ralphm
cc__ralphm cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_3"></a>|@ralphm commented|
|-|-|

(In [33867]) Branching to 'xish-epytext-5555'

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_4"></a>|@ralphm set owner to @ralphm<br>@ralphm set status to `assigned`|
|-|-|

Apologies for taking over this ticket. I wasn't aware that it was meant as a way to give our new contributors some easy tickets.

I removed the references to build-in types like `str`, `unicode`, etc., and to external modules.

I left the other warnings for what they are. Apparently, there are quite some issues with the object resolution within Pydoctor itself:

 * Links to (local and external) module attributes are not resolved.
 * In some cases, two modules documenting things that are in the other module, don't get resolved.
 * Local class attributes sometimes don't resolve (like `logTraffic` in `StreamManager`).

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_5"></a>|@ralphm removed owner<br>@ralphm set status to `new`|
|-|-|

I think this should be ready now. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_6"></a>|@ralphm commented|
|-|-|

References to related Pydoctor issues: lp:685376 and lp:957014.

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_7"></a>|@ralphm commented|
|-|-|

And lp:712919.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra set owner to @ralphm|
|-|-|

Replying to [ralphm](#note_4):
> Apologies for taking over this ticket. I wasn't aware that it was meant as a way to give our new contributors some easy tickets.
> 
> I removed the references to build-in types like `str`, `unicode`, etc., and to external modules.
> 
> I left the other warnings for what they are. Apparently, there are quite some issues with the object resolution within Pydoctor itself:
> 
>  * Links to (local and external) module attributes are not resolved.
>  * In some cases, two modules documenting things that are in the other module, don't get resolved.
>  * Local class attributes sometimes don't resolve (like `logTraffic` in `StreamManager`).

Thanks for fixing those. Could you also fix the other coding standard stuff like putting docstrings on 3 lines (without the current prefixing whitespace):
```
"""
Generic Streaming XML protocol handler.
```

The news file can be an empty .misc file, I doubt users will care we fixed an api doc issue (cause there are many).

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_9"></a>|@ralphm set status to `assigned`|
|-|-|

Replying to [thijs](#note_8):
> [..] Could you also fix the other coding standard stuff [..]

It is better to do these changes as part of another ticket, as exarkun rightly mentioned in #twisted.

> The news file can be an empty .misc file, I doubt users will care we fixed an api doc issue (cause there are many).

Fixed.

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_10"></a>|@ralphm set status to `closed`|
|-|-|

(In [33882]) Merge xish-epytext-5555: Fix epytext warnings in xish and jabber modules.

Author: ralphm
Reviewer: thijs
Fixes: [#5555](https://github.com/twisted/twisted/issues/5555)
